var/spack/repos/builtin/packages/lzo/package.py

```diff
@@ -37,15 +37,8 @@ class Lzo(Package):
     version('2.06', '95380bd4081f85ef08c5209f4107e9f8')
     version('2.05', 'c67cda5fa191bab761c7cb06fe091e36')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            '--prefix={0}'.format(prefix),
+    def configure_args(self):
+        return [
             '--disable-dependency-tracking',
             '--enable-shared'
         ]
-        configure(*configure_args)
-        make()
-        if self.run_tests:
-            make('check')
```

Comment:
@davydden Yes, as they are both present. But they will run only if you give the option : 
```
 spack install --run-test lzo
```
In general the default behavior I encoded is to search for `check` and `test` targets in `AutotoolsPackage`s and only for `test` in `CMakePackage`s . This seems to me to follow what it is usually done in the two build-systems.